### ISSUE TITLE
Partclone `0.2.91` > `0.3.6` ntfs regression

### ISSUE THREAD

**Issue by porsager** [Thomas-Tsai/partclone#109]
I'm trying to restore ntfs partitions made using partclone `0.2.91` with `0.3.6`, but when ntfsresize runs afterwards the following error comes up.

```
ntfsresize -f -f /dev/mmcblk1p3
ntfsresize v2016.2.22AR.2 (libntfs-3g)
ntfs_mst_post_read_fixup_warn: magic: 0x666f736f   size: 1024    usa_ofs:11892    use_count: 26967: Invalid argument
Record 0 has no FILE magic (0x666f736f)
Failed to load $MFT: Input/output error
ERROR(5): Opening '/dev/mmcblk1p3' as NTFS failed: Input/output error
NTFS is inconsistent. Run chkdsk /f on Windows then reboot it TWICE!
```

There is also an issue open in clonezilla, but my suspicion is that it might be an issue with partclone. https://github.com/stevenshiau/clonezilla/issues/31#issuecomment-381364966

**Comment by porsager** [comment]
Yes, 64bit

**Comment by neoclust** [comment]
can you try clonezilla 32bit ?  just to make sure we have the same behaviour.
For me all is OK on 32bit clonezilla

**Comment by porsager** [comment]
Yeah, I'll try to give that a shot, but I'm unfortuntely not able to until next week..

**Comment by neoclust** [comment]
some news about this ? i can't clone/restore any image using partclone 0.3.x :(

**Comment by porsager** [comment]
Did you have any luck with 32bit clonezilla btw? (32bit doesn't work for me either)

**Comment by neoclust** [comment]
no i only tried on 64bit.

And the issue is still valid for me.


partclone 0.2 -> 0.2  OK
partclone 0.3 -> 0.3 OK
partclone 0.2 -> 0.3 -> NOK

**Comment by Thomas-Tsai** [comment]
This issue has been inactive for a long time.

As our project has evolved since then, we assume this is no longer relevant or may have been resolved indirectly.

I'm closing this to keep our issue list manageable. If you find that this problem still persists, please leave a comment and we will happily reopen it. Thanks for your understanding!
